src/core/exec-invoke.c

```diff
@@ -2236,45 +2236,61 @@ static int bpffs_prepare(
 
         r = socketpair(AF_UNIX, SOCK_SEQPACKET|SOCK_CLOEXEC, 0, pipe_fds);
         if (r < 0)
-                return r;
+                return log_error_errno(errno, "Failed to create socket pair: %m");
 
         r = pidref_safe_fork("(sd-bpffs)", FORK_RESET_SIGNALS|FORK_DEATHSIG_SIGKILL, ret_pid);
         if (r < 0)
-                return r;
+                return log_error_errno(errno, "Failed to fork: %m");
```

Comment:
`errno` -> `r`

Reply:
Hi, I'm adding logs here just to debug why PrivateBPF is working locally and failing in the CI.
But if you want I can keep them, with better error messages.